torch/distributed/device_mesh.py

```diff
@@ -544,10 +548,16 @@ def _init_process_groups(self):
                         # We temporarily revert the re-use subgroup, since it breaks two internal tests.
                         # Temporarily reverting to resolve test timeout while root-causing.
                         # TODO: Add two tests to cover internal tests scenarios and re-enable reuse subgroup if exists.
+                        group_desc = (
+                            f"mesh_dim_{self.mesh_dim_names[dim]}"
```

Comment:
Thanks for adding that.

Reply:
Hmmm, actually, can we name it "mesh_{self.mesh_dim_names[dim]}" or just "{self.mesh_dim_names[dim]}". Also, for the case when self.mesh_dim_names is not given, do you think it's better that we store a desc like "mesh_dim_{dim}".

because when debugging, user can just filter out to certain PGs using this, so I guess something like "mesh_tp", "mesh_pp" or "mesh_dim_0" are simple enough? WDYT?

Reply:
Ye. Sounds good. Nice catch. We should still keep the group_desc when mesh_dim_names is None. Let me update this.